g11n-ws/modules/md-service-i18n-l3/src/main/java/com/vmware/vip/core/messages/service/product/ProductService.java

```diff
@@ -166,6 +171,19 @@ private DropVersionDTO getDropVersion(DropVersionDTO versioninfo,JSONObject jo,
 						+ componentMessagesDTO.getVersion(), e);
 			}
 		}
+		com.vmware.vip.core.messages.service.multcomponent.TranslationDTO multComp = new com.vmware.vip.core.messages.service.multcomponent.TranslationDTO();
+		multComp.setProductName(productName);
+		multComp.setVersion(version);
+		multComp.setLocales(getSupportedLocaleList(productName, version));
+		multComp.setComponents(getComponentNameList(productName, version));
+		String multKey = CachedKeyGetter.getMultiComponentsCachedKey(multComp);
+		try {
+			TranslationCache3.deleteCachedObject(CacheName.MULTCOMPONENT, multKey, com.vmware.vip.core.messages.service.multcomponent.TranslationDTO.class);
+		} catch (VIPCacheException e) {
+			// TODO Auto-generated catch block
+			throw new L3APIException(
+					"MULTCOMPONENT cache occurs error when update translation.", e);
+		}
```

Comment:
Why delete the cache？

Reply:
Delete cache content that have updated translation

Reply:
This use the deleteCachedObject method,  not delete cache